code/disk_wrapper_ioctl.h

```diff
@@ -14,6 +14,105 @@
 #define COW_BRD_RESTORE_SNAPSHOT  0xff08
 #define COW_BRD_WIPE              0xff09
 
+// Defines that are separate from the kernel because these values aren't stable.
+// Based on 4.4 kernel flags.
+//
+// TODO(ashmrtn): These should be narrowed down to just the flags that we
+// actually care about.
+enum flag_shifts {
```

Comment:
Could we get short comments about some of the flags that are not obvious?

Reply:
I'll try, though the kernel doesn't really say what most of the flags are either. As the TODO says, we should probably try to slim this down a bit. I included the full complement because I'm not sure what @SoujanyaPonnapalli is using now in her work. With some discussion, we could probably reduce this list.